tests/py/test_sync_npm.py

```diff
@@ -85,21 +83,6 @@ def test_even_deletes_package_with_linked_team(self):
         sync_npm.consume_change_stream(self.change_stream(docs), self.db)
         assert self.db.one('select * from packages') is None
 
-
-    def test_picks_up_with_last_seq(self):
```

Comment:
This test currently only tested our test function (`change_stream(self, changes)`). The job of picking up from `last_seq` now belongs to `production_change_stream`, which internally calls the CouchDB library, and hence we don't need to test it.